spec/spec_helper.rb

```diff
@@ -44,7 +45,7 @@
   # inherited by the metadata hash of host groups and examples, rather than
   # triggering implicit auto-inclusion in groups with matching metadata.
   config.shared_context_metadata_behavior = :apply_to_host_groups
-
+  config.include FactoryBot::Syntax::Methods
```

Comment:
I added this line in to the `rails_helper.rb` that's where I have always had we probably don't want it in both places just so it does not cause a conflict with itself. 

Do you mind removing it from one or the other file? I don't care if it is in this one ore the `rails_helper` if you prefer one over the other.

Reply:
I removed the rails_helper line!